contracts/examples/DXswapRelayer.sol

```diff
@@ -188,29 +190,21 @@ contract DXswapRelayer {
     function executeOrder(uint256 orderIndex) external {
         Order storage order = orders[orderIndex];
         require(orderIndex <= orderCount && orderIndex != 0, 'DXswapRelayer: INVALID_ORDER');
```

Comment:
Since you've changed the behavior of `_OrderIndex()`, the `&& orderIndex != 0` bit of this assertion should go away.

Reply:
@philhofer just stumbled upon that on the test as well. Thanks for checking!